.github/workflows/workflow.yml

```diff
@@ -11,6 +11,11 @@ on:
 permissions:
   contents: read # to fetch code (actions/checkout)
 
+# cancel previous test builds if new commit is pushed.
+concurrency:
+  group: workflow-master-${{ github.event_name }}-${{ github.event.number }}
+  cancel-in-progress: true
```

Comment:
This is an unrelated change but it might help reduce wait time when you push a fix but the previous build is still running.

This is how it looks like: 

![image](https://github.com/rq/rq/assets/9079960/732b85f7-9c96-4cb1-ba42-f856acb4894e)

Reply:
Thanks, this is useful!